pkg/aggregator/sender_test.go

```diff
@@ -40,46 +40,51 @@ func initSender(id check.ID, defaultHostname string) (s senderWithChans) {
 	return s
 }
 
+func testDemux() *AgentDemultiplexer {
+	opts := DefaultDemultiplexerOptions(nil)
+	opts.DontStartForwarders = true
+	demux := initAgentDemultiplexer(opts, defaultHostname)
+	return demux
+}
+
 func TestGetDefaultSenderReturnsSameSender(t *testing.T) {
-	// this test IS USING globals (demultiplexerInstance holding the senders)
+	// this test not using anything global
```

Comment:
Is this message still relevant as most tests do not use anything global?

Reply:
I think I've left a few comments still using globals in other files.
Seeing the history of people fighting with globals in tests, I prefer to keep them for a while if you don't mind.